Lib/textwrap.py

```diff
@@ -125,7 +125,8 @@ def __init__(self,
                  tabsize=8,
                  *,
                  max_lines=None,
-                 placeholder=' [...]'):
+                 placeholder=' [...]',
+                 text_len=len):
```

Comment:
Note: because these are keyword-only params, text_len doesn’t have to be added at the end.

Reply:
I am not entirely sure what you mean. Is there something I should change or is that just a general remark?

Reply:
You could move text_len before of after max_lines if you thought it makes more sense to have them together.  This would have the effect of minimizing the diff (and preserving history for the line that adds `placeholder`) which is a very minor but nice thing 🙂